frontend/src/app/home/homepage.component.html

```diff
@@ -1,6 +1,6 @@
 <volontulo-banner></volontulo-banner>
-<div class="container">
+<section class="offer-list">
```

Comment:
can we stick with `volontulo-` prefix for class here? Moreover that homepage-offer-list (we have other view for offer lists also).

Reply:
I will do that for now, however I don't think it is a good idea to use volontulo prefix everywhere. I do understand a reason for homepage- prefix, but not for volontulo- . The issue is more connected to how we use bootstrap in our project, which makes a HTML markup really messy. We can talk about that on video. I'll try to catch you next Wednesday.